helm/chaos-mesh/templates/controller-manager-deployment.yaml

```diff
@@ -22,8 +22,9 @@ spec:
         app.kubernetes.io/name: {{ template "chaos-mesh.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
         app.kubernetes.io/component: controller-manager
-    {{- with .Values.controllerManager.podAnnotations }}
       annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
Use `hack/update_install_script.sh` to update `install.sh`

Reply:
Fixed